liveblog.php

```diff
@@ -897,7 +897,7 @@ private static function add_default_plupload_settings() {
 	 */
 	private static function get_entries_endpoint_url() {
 		if ( self::use_rest_api() ) {
-			$url = trailingslashit( trailingslashit( WPCOM_Liveblog_Rest_Api::build_endpoint_base() ) . self::$post_id );
+			$url = WPCOM_Liveblog_Rest_Api::build_endpoint_base() . self::$post_id . '/';
```

Comment:
When the javascript goes to use that `url` it expects it to already have the trailing slash, as you can see in `js/liveblog.js` (eg. `url += 'entries/‘;`).
This is just the base path and something else will always be appended in order to build a valid endpoint URL.

Whereas setting `$endpoint_url` in `admin_enqueue_scripts()` for example is building the full URL with nothing that needs to be appended, and so there’s no trailing slash in that case.

Are there other endpoints that you were referring to?